src/Containers.jsx

```diff
@@ -76,13 +78,16 @@ const ContainerActions = ({ container, healthcheck, onAddNotification, version,
 
     const stopContainer = (force) => {
         const args = {};
+        if (force)
+            args.t = 0;
+
+        utils.debug(container.isSystem, "stopContainer", container.Id, "args", JSON.stringify(args));
 
         setActionsKebabOpen(false);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230705-163420-8438f4c3-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)